packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -639,9 +654,32 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
     }
 
     // Add "Actions" column
-    if (editItem || contentEditor.enabled !== false) {
+    if (editItem || contentEditor.enabled !== false || hasInsightsFlyout) {
       const actions: EuiTableActionsColumnType<T>['actions'] = [];
 
+      if (hasInsightsFlyout) {
+        actions.push({
+          name: (item) => {
+            return i18n.translate('contentManagement.tableList.listing.table.viewStatsActionName', {
+              defaultMessage: 'View {itemTitle} stats',
+              values: {
+                itemTitle: get(item, 'attributes.title'),
+              },
+            });
+          },
+          description: i18n.translate(
+            'contentManagement.tableList.listing.table.viewStatsActionDescription',
+            {
+              defaultMessage: 'View stats',
```

Comment:
```suggestion
              defaultMessage: 'View insights',
```
Just to align the wording as this new flyout uses the term "insights"

Reply:
I agree. If we go with a different word than `insights`, we should replace the various occurrences to stay consistent, including in the flyout.

Reply:
@teresaalvarezsoler @florent-leborgne I like the word `Usage` though at that point I'm thinking does the icon (stats) still make sense? do we really need a separate flyout? what if simply use the `Dashboard details` flyout and place all this info under the `Activity` section?

<img width="687" alt="image" src="https://github.com/user-attachments/assets/1d2dfe06-f224-4534-a12e-4d00786cdc64">

Reply:
@andreadelrio yes, I think it makes things simpler, but I don't have a strong opinion. It's your call and also considering the effort @Dosant needs to do to change it now...

Reply:
- Re: using the stats icon with the word `Usage` (or `View usage` for the tooltip), I think the combination makes sense and is a good hint on the info behind it.

- 2 cents on the IA: I think it would be simpler to have only 1 flyout unless there are plans to expand the Usage/Stats info later, but this wouldn't be a game changer as compared to having 2 flyouts. For example:
**Dashboard details**
      ∟Metadata (including `created by`/`last edited by` info, and not necessary to use the term 'metadata')
      ∟Usage
But given the current layout, this would push the graph quite further down in the flyout, which is maybe not ideal.

Reply:
About using a single flyout: I'd also consider the Dashboard Details flyout is an editable form where you can edit info and press save from the footer. The created by / edit by / views are not editable. If we'd move everything under a single flyout, then the placement of the "Update" button should probably be re-considered 

Reply:
Thanks everyone for your input. I keep going back to thinking the `iInCircle` icon is the best suited one for this information. For that reason, I think we should stick with a single flyout with the **Activity** subheading. The tooltip that we have should be enough to explain that that data is non-editable, no need to move the `Update` button. This would be the result for Editor and Viewer roles:

<img width="1235" alt="image" src="https://github.com/user-attachments/assets/be30c555-03d8-45a5-a88d-30435aa5db2e">

Reply:
Thanks, I changed to a single flyout 👍